schedule/sles4sap/sles4sap_gnome_saptune.yaml

```diff
@@ -8,6 +8,7 @@ vars:
   BOOT_HDD_IMAGE: '1'
   NODE_COUNT: '1'
   TEST_CONTEXT: 'OpenQA::Test::RunArgs'
+  PUBLIC_CLOUD_SSH_CONFIG: 'publiccloud/ssh_config_sap'
```

Comment:
Heh. I thought since saptune tests are using the SSH terminal connection/tunnel thingy, that this was not necessary here, but if it works, awesome!